core/src/main/resources/application.yml

```diff
@@ -853,9 +853,13 @@ crn:
   region: us-west-1
 
 existingstackpatcher:
-  intervalhours: 24
+  intervalInHours: 24
   maxInitialStartDelayInHours: 2
-  enabled: true
+  enabled:
+    - UNBOUND_RESTART
+    - LOGGING_AGENT_AUTO_RESTART
```

Comment:
thanks for the patch i think you can disable the logigng agent onces + metering azure metadata

Reply:
I have disabled all stack patches